docs/releaseNotes.yml

```diff
@@ -86,6 +86,19 @@ items:
           This allows the healthcheck server to be configured to use IPv6-only k8s environments. 
           (Thanks to <a href="https://github.com/TimonOmsk">Dmitry Golushko</a>!).
 
+      - title: Upgrade to Envoy 1.24.1
+        type: feature
+        body: >-
```

Comment:
perhaps you should link to Tenshin's new docs for OTel as well

Reply:
That isn't merged yet, so left it off. My plan was to do it to get it merged on getambassador.io.